tools/src/bin/uurlog/invoice.rs

```diff
@@ -18,7 +18,19 @@ pub struct InvoiceOptions {
 	/// The period to create an invoice for.
 	#[structopt(long)]
 	#[structopt(value_name = "YYYY[-MM[-DD]]")]
-	period: PartialDate,
+	period: Option<PartialDate>,
```

Comment:
Hmm, I do kinda want period or start/end date to be mandatory. Making an invoice for the full log is not something that really makes sense.

Reply:
How about making start-date require end-date, end-date require start-date and adding `period` and `start-date` to a group to make one of them mandetory?